Header header

string MASK_ALL=all
string MASK_ACTIVE=active

string mask
int8 num_drones

# size num_drones
float64[] x
float64[] y
float64[] z
float64[] roll
float64[] pitch
float64[] yaw